kv_store/src/bank/multi/transaction.rs

```diff
@@ -43,4 +45,15 @@ impl<DB: Database, SK: StoreKey> MultiBank<DB, SK, TransactionStore<DB, SK>> {
             store.append_block_cache(other.kv_store_mut(sk))
         }
     }
+
+    pub fn take_block_cache(
```

Comment:
It seems like we do many extra clones here. Do we? Maybe it's not an issue, I just want to understand is it okay or not.

Reply:
This is not a clone (except SK which is simple enum), but you're right that all this stuff needs to be optimized at some point.